packages/webapp/src/members/transactions.ts

```diff
@@ -0,0 +1,79 @@
+import { Asset, assetToString } from "_app";
+import { edenContractAccount, tokenConfig } from "config";
+
+import { DistributionAccount } from "delegates/interfaces";
+
+export const withdrawAndTransfer = (
+    authorizerAccount: string,
+    totalWithdrawal: Asset,
+    destinationAccount: string,
+    memo: string,
+    distributions?: DistributionAccount[]
+) => {
+    if (!totalWithdrawal) {
+        throw new Error("The current available balance to withdraw is zero");
+    }
+
+    const isThirdPartyTransfer = destinationAccount !== authorizerAccount;
+    const amount = assetToString(totalWithdrawal, totalWithdrawal.precision);
+
+    const authorization = [
+        {
+            actor: authorizerAccount,
+            permission: "active",
+        },
+    ];
+
+    const sweepAvailableDistributions =
```

Comment:
question: so, by design, the idea is to sweep everything and only transfer the specified amount, correct?

Reply:
Exactly. This is the stopgap solution just to get this out there. We'll do a more complete enhanced-funds-management feature in the future.